### ISSUE TITLE
2.7: issue with latest RandomLib

### ISSUE THREAD

**Issue by remicollet** [zendframework/zend-math#23]
With RandomLib (and without mcrypt), running zend-form test suite:

`PHP Fatal error:  Class Zend\Math\Source\HashTiming contains 1 abstract method and must therefore be declared abstract or implement the remaining methods (RandomLib\Source::isSupported) in /usr/share/php/Zend/Math/Source/HashTiming.php on line 111
`

**Comment by remicollet** [comment]
Notice also raised during zend-math test suite, see (Fedora QA)
https://apps.fedoraproject.org/koschei/package/php-zendframework-zend-math?collection=f26

**Comment by ezimuel** [comment]
@remicollet [ircmaxell/RandomLib](https://github.com/ircmaxell/RandomLib/blob/master/lib/RandomLib/Source.php) added the method `isSupported` to the interface `RandomLib\Source`. To fix this we need to merge #24 as new release 2.7.1. @weierophinney what do you think?

**Comment by weierophinney** [comment]
Let's wait and see if Anthony addresses the BC break, first; if so, we can potentially exclude that release from our supported versions list.

**Comment by remicollet** [comment]
Anthony have answered on PR #24

**Comment by tasmaniski** [comment]
Yep, I have same issue. Does anyone found hack for this fatal error?

**Comment by reinfi** [comment]
When will this be fixed?

**Comment by weierophinney** [comment]
This repository has been closed and moved to laminas/laminas-math; a new issue has been opened at https://github.com/laminas/laminas-math/issues/1.
